### ISSUE TITLE
Setuptools ignored

### ISSUE THREAD

**Issue by OrangeDog** [jgonggrijp/pip-review#13]
On Windows Python 2.7.12 you get this peculiar output:

```
> pip list --outdated
cassandra-driver (3.4.0) - Latest: 3.7.0 [sdist]
setuptools (20.10.1) - Latest: 28.6.1 [wheel]

> pip-review
)assandra-driver==3.7.0 is available (you have 3.4.0
```

I then proceeded to successfully update cassandra-driver, but it doesn't appear to be able to see setuptools at all.

```
>pip list --outdated
setuptools (20.10.1) - Latest: 28.6.1 [wheel]

>pip-review
Everything up-to-date
```

**Comment by jgonggrijp** [comment]
Regarding the omission of `setuptools`, that reminds me of the omission of `pip` itself which appeared in an older issue (linked above). Fixing it on my own is not high enough on my priority list, but pull requests are welcome.

Regarding the misplaced `(` in the console output: is that happening consistently in Windows Py 2? If so, please open a new ticket and specify your terminal emulator (e.g. Win 10 Command Prompt), locale settings and version of pip-review.

**Comment by jgonggrijp** [comment]
I now realize that this issue was closed by accident. Reopening.

**Comment by jgonggrijp** [comment]
@k4rtik pointed out that the problem is also occurring in macOS. It probably applies to every Python version on every platform, hence the title change.

**Comment by jgonggrijp** [comment]
On 21. December I said this issue was closed by accident (see above). I was wrong; the issue was actually solved by @xoviat in ea283bb. Closing forever now. Sorry for the confusion.
